From ad578d3affde3456504b580b12c3a934358f99b9 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@inbox.ru>
Date: Mon, 26 Apr 2010 21:36:40 +0300
Subject: [PATCH] fixed build failure: 'Main did not exported main'

src/Main.hs:1:0:
    The main function `main' is not exported by module `Main'

Signed-off-by: Sergei Trofimovich <slyfox@inbox.ru>
---
 src/Main.hs |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/src/Main.hs b/src/Main.hs
index d098634..d5adcec 100644
--- a/src/Main.hs
+++ b/src/Main.hs
@@ -1,5 +1,4 @@
-module Main
-    () where
+module Main (main) where
 
 import Graphics.UI.GLUT
 import System.Exit
-- 
1.6.4.4

